functions/src/models/base-resource-object.test.ts

```diff
@@ -48,6 +48,21 @@ const invalidMemberAccessRules: AccessRule[] = [
   }
 ];
 
+const validContextAccessRules: AccessRule[] = [
+  {
```

Comment:
probably should add a user rule to this array since that will be the most common setup of AccessRule.

Reply:
Done in the last commit.